service/controller.go

```diff
@@ -2608,8 +2678,10 @@ func (s *service) systemProbe(_ context.Context, array *ArrayConnectionData) err
 		}
 	}
 
+	Log.Printf("Login to VxFlexOS Gateway, system=%s, endpoint=%s, user=%s\n", systemID, array.Endpoint, array.Username)
```

Comment:
VxFlexOS -> PowerFlex

Reply:
This might actually have been leftover from @falfaroc 's changes. 
I'll remove unless he says otherwise.

Reply:
This has been around from the start, good to start converting.
Old: Vxflexos.
Best: PowerFlex.